.github/workflows/release-build.yml

```diff
@@ -10,7 +10,7 @@ permissions:
 jobs:
   build:
     name: Push Release
-    runs-on: ARM64
+    runs-on: ubuntu-24.04-arm
```

Comment:
same as for build, I'd keep this using the ARM64 machine because this cross-compiles